doc/CHEATSHEET.md

```diff
@@ -2767,3 +2767,38 @@ try (KnativeClient kn = new DefaultKnativeClient()) {
     kn.services().inNamespace("default").createOrReplace(service);
 }
 ```
+
+#### Logging
+Using logging-interceptor:
+
+- Configure OkHTTP logging:
+``` 
+Logger reqLogger = LoggerFactory.getLogger(HttpLoggingInterceptor.class); 
+```
+ if (reqLogger.isTraceEnabled()) { 
+     HttpLoggingInterceptor loggingInterceptor = new HttpLoggingInterceptor(); 
+     loggingInterceptor.setLevel(HttpLoggingInterceptor.Level.BODY); 
+     httpClientBuilder.addNetworkInterceptor(loggingInterceptor); 
+ } 
```

Comment:
Is this code from [HttpClientUtils ](https://github.com/fabric8io/kubernetes-client/blob/master/kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/HttpClientUtils.java#L146)in kubernetes-client? Why do we need this here?